src/magic/model/MagicPermanent.java

```diff
@@ -1168,6 +1182,10 @@ public boolean isAura() {
         return isEnchantment() && hasSubType(MagicSubType.Aura);
     }
 
+    public boolean isSaga() {
```

Comment:
move this to a default method of IRenderableCard as it is the same between MagicPermanent and MagicCardDefinition

Reply:
Should I also move other methods like `isAura` there in later PR too?